package.json

```diff
@@ -12,11 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
     "scripts": {
-        "start": "webpack-serve --config ./webpack.serve.config.js --open --port 8000",
+        "start": "set NODE_OPTIONS=--openssl-legacy-provider && webpack-serve --config ./webpack.serve.config.js --open --port 8000",

```

Comment:
I would use https://www.npmjs.com/package/dotenv in dev and add it to the package .env file.i think it's react version that makes this required